pkg/config/expand.go

```diff
@@ -131,6 +131,7 @@ func (bp *Blueprint) expandGroups() error {
 func (bp Blueprint) expandGroup(gp groupPath, g *Group) error {
 	var errs Errors
 	bp.expandBackend(g)
+	bp.expandProviders(g)
```

Comment:
Wrap it into `if grp.Kind() is Terraform`

Reply:
Done.